tests/node/rpc/messages/GetVehicleDataResponseTests.js

```diff
@@ -68,12 +70,14 @@ describe('GetVehicleDataResponseTests', function () {
         const testHandsOffSteering = rpcMessage.getHandsOffSteering();
         const testWindowStatus = rpcMessage.getWindowStatus();
         const testGearStatus = rpcMessage.getGearStatus();
+        const testSeatOccupancy = rpcMessage.getSeatOccupancy();
 
         // Valid Tests
         Validator.assertEquals(this.stabilityControlsStatus, testStabilityControlsStatus);
         Validator.assertEquals(Test.GENERAL_BOOLEAN, testHandsOffSteering);
         Validator.assertEquals([Test.GENERAL_WINDOW_STATUS], testWindowStatus);
         Validator.assertEquals(this.gearStatus, testGearStatus);
+        Validator.assertEquals(Test.GENERAL_SEAT_OCCUPANCY, testSeatOccupancy);
```

Comment:
On line 80 are brackets ( [] ) needed around `Test.GENERAL_SEAT_OCCUPANCY` similar to line 76 for `Test.GENERAL_WINDOW_STATUS`?

Reply:
No, because seatOccupancy is not an array [setSeatOccupancy](https://github.com/smartdevicelink/sdl_javascript_suite/blob/edf43e805626a3f5de97bce33c3e5ef7a4c064ba/lib/js/src/rpc/messages/GetVehicleDataResponse.js#L760), compared to windowStatus which is indeed an array 
[setWindowStatus](https://github.com/smartdevicelink/sdl_javascript_suite/blob/edf43e805626a3f5de97bce33c3e5ef7a4c064ba/lib/js/src/rpc/messages/GetVehicleDataResponse.js#L720)